lib/ttt.ex

```diff
@@ -53,7 +51,7 @@ defmodule TTT do
   end
 
   def get_input(turn) do
-    move = IO.gets "'#{turn}', please enter a number from 1 to 9 only:"
+    move = IO.gets "'#{turn}', please enter a number from 1 to 9 only: "
     case not valid_input?(move) do
```

Comment:
Why are you using `case not valid_input?` rather than just `case valid_input`? That makes the code harder to read.